src/sdk/k4a.c

```diff
@@ -258,7 +263,6 @@ void k4a_device_close(k4a_device_t device_handle)
     }
     k4a_device_t_destroy(device_handle);
     allocator_deinitialize();
-    assert(allocator_test_for_leaks() == 0);
```

Comment:
If you remove this, then are we checking for leaks anymore?

Reply:
This assert fails almost 100% of the time when using the managed wrapper. It's only valid to call allocator_test_for_leaks if you know that all the k4a objects have been dereferenced, which isn't the case during k4a_device_close. In fact we almost certainly should add more tests that we can close the device before dereferencing the images.

Reply:
Gotcha, agreed, I was thinking this would be another case were we might want to just run this once just before the binary unloads, or just drop it all together. Calling it at the end of UT's might be enough to catch regression in the SDK.